Scripts/SetAndHandleEmpty/SetAndHandleEmpty.yml

```diff
@@ -0,0 +1,41 @@
+commonfields:
+  id: SetAndHandleEmpty
+  version: -1
+name: SetAndHandleEmpty
+fromversion: 4.5.0
+type: python
+tags:
+- Utility
+comment: Checks if value exists in context. If the value exists it will be set in context,
```

Comment:
Checks if the specified value exists in context. If the value exists, it will be set in context, otherwise no value will be set it context.

@David-BMS Is this still technically accurate?

Reply:
It sounds confusing but the intention is that the script checks for a content value in one location and if it exists sets it in another location which is needed for the context outputs